python/pdo/submitter/submitter.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Intel Corporation
+# Copyright 2020 Intel Corporation
```

Comment:
Nit: I don't think we need to change the year here.

Reply:
thank you for the comment. this is an entirely new file though we are retaining the name.